core/src/wsv.rs

```diff
@@ -173,6 +173,20 @@ impl<W: WorldTrait> WorldStateView<W> {
             .collect()
     }
 
+    /// Calculates network TPS.
+    ///
+    /// `None` means that transactions can not yet be calculated.
+    #[allow(clippy::float_arithmetic, clippy::cast_precision_loss)]
+    pub fn transactions_per_second(&self) -> Option<f64> {
+        let start_time_ms = self.blocks.iter().next()?.as_inner_v1().header.timestamp;
```

Comment:
It may be trivial, but
there may be a good amount of time between committing genesis and starting service for real users